src/Switches.cpp

```diff
@@ -597,16 +597,17 @@ bt::Node::Ptr Switches::tacticSwitch(std::string name, bt::Blackboard::Ptr prope
             }
             },
             {"calibration_tactic", {
-                    {"c1", robotType::CLOSE_TO_BALL},
-                    {"c2", robotType::RANDOM},
-                    {"c3", robotType::RANDOM},
-                    {"c4", robotType::RANDOM},
-                    {"c5", robotType::RANDOM},
-                    {"c6", robotType::RANDOM},
-                    {"c7", robotType::RANDOM},
-                    {"c8", robotType::RANDOM}
-            }
+                {"c1", robotType::CLOSE_TO_BALL},
```

Comment:
Honestly this whole switches.cpp file is kinda questionable, but whatever, there might not be a better way.